Add db setup instructions (#14)
rlucas7/code-searcher: README.md

5. After running the makefile all target, via `make all` in the same directory as the makefile,
one should have an executable `*.dylib` file running locally-assuming the makefile executed fully. For instance the dylib file was under dist and the makefile will put the sqlite3 executable under the same directory. The path looks like `sqlite-vec/dist/vec0.dylib` .

One can also see this referenced in the `config.py` module in the flask app for the project, under the `_SQLITE_VEC_DLL_PATH` value in `config.py`.

6. Confirm that the sqlite3 build can use the extension. The make build generates the executable sqlite3 inside the dist directory. We can test that the vector extension was built successfully via the examples in the readme for the extension https://github.com/asg017/sqlite-vec?tab=readme-ov-file#sample-usage, before proceeding to flask setup. We want to confirm that sqlite can use the extension on your machine regardless of the flask& python setup/configs.

After running `make all`, to run the above sample commands one must start REPL from the root of sqlite-vec repo with:

```bash
./dist/sqlite3
```

> Then ca run sample-usage line by line from here https://github.com/asg017/sqlite-vec?tab=readme-ov-file#sample-usage in REPL starting with:

Reviewer: ```suggestion
Then run sample-usage line by line from here https://github.com/asg017/sqlite-vec?tab=readme-ov-file#sample-usage in REPL starting with:
```

suggest deleting a couple stray characters

```bash
.load ./dist/vec0
```

> 

Reviewer: ```suggestion
if you are able to run the example SQL lines from the linked file then you have successfully build the executable for sqlite and the vector extension. 
then next step is to setup the app to run locally, the app uses the vector extension. 
```

> Following steps were tested for Python 3.11 (MacOS M1 and intel).

Reviewer: ```suggestion
Following steps were tested for Python 3.11 (using Mac M1 and Mac intel chips).
```

slight rephrasing
